packages/firestore/lite/src/api/components.ts

```diff
@@ -88,3 +83,18 @@ export function removeComponents(firestore: FirebaseFirestore): void {
     datastore.terminate();
   }
 }
+
+export function makeDatabaseInfo(
+  databaseId: DatabaseId,
+  persistenceKey: string,
+  settings: Settings
+): DatabaseInfo {
+  return new DatabaseInfo(
+    databaseId,
+    persistenceKey,
+    settings.host ?? DEFAULT_HOST,
+    settings.ssl ?? DEFAULT_SSL,
+    /* forceLongPolling= */ false,
+    /* forceAutoDetectLongPolling= */ true
```

Comment:
Why not consult settings for this?

Reply:
I originally chose not to pull in FirestoreSettings into firestore-exp since it included a lot of validation code. This is no longer the case so I was able to use FirestoreStettings here.